lib/generators/spotlight/templates/config/initializers/translation.rb

```diff
@@ -5,19 +5,20 @@
 
 Translation = I18n::Backend::ActiveRecord::Translation
 
-ActiveSupport::Reloader.to_prepare do
-  if Translation.table_exists?
-    ##
-    # Sets up the new Spotlight Translation backend, backed by ActiveRecord. To
-    # turn on the ActiveRecord backend, uncomment the following lines.
+# Don't allow initializer to break if DB doesn't exist yet
+# see: https://github.com/projectblacklight/spotlight/issues/2133
+if ENV['SKIP_TRANSLATION'].blank?
```

Comment:
this is a solution that @mcritchlow figured out for the Starlight fork: https://gitlab.com/surfliner/surfliner/-/commit/aa0a8183246326e3317997c7f19fe2285087b321